src/translate.rs

```diff
@@ -30,6 +37,13 @@
 //!             from_glib_none(title)
 //!         }
 //!     }
+//!     fn create_bool(value: gboolean) -> Variant {
+//!         unsafe {
+//!             let variant = glib_sys::g_variant_new_boolean(value);
+//!             // g_variant_new_boolean has `transfer none`
```

Comment:
This comment can be added to previous example: // gtk_window_get_title has `transfer none` for return type

Reply:
I wanted to do that, but the docs have no annotation other than `[nullable]`: https://developer.gnome.org/gtk3/stable/GtkWindow.html#gtk-window-get-title

and it's a string and not a GObject, so I thought it's good to have a string example too.

Reply:
GVariant also not GObject.
Ok, I vote up for current version.

Reply:
There are some defaulting rules for cases where there are no annotations, that's why `gtk_window_get_title()` has no explicit annotations. It's somewhere on the GObject-Introspection wiki.